webpack.config.js

```diff
@@ -32,7 +33,34 @@ module.exports = (env) => ({
             options: {
               svgo: true,
               svgoConfig: {
-                plugins: [{ removeAttrs: false, prefixIds: false }]
+                plugins: [
+                  {
+                    removeAttrs: false,
+                    prefixIds: false
+                  },
+                  {
+                    addClassNamesPlugin: Object.assign(
+                      {},
+                      addClassNamesPlugin,
+                      {
+                        params: {
+                          rules: [
+                            {
+                              attribute: 'fill',
```

Comment:
I would configure the same classname to be applied to both, fill and stroke and to drop it from the name itself. This way we would have base and highlight classes per icon type.

Reply:
Is this like you meant?

```js
rules: [
  {
    attribute: 'fill|stroke',
    value: '#A5ADB1',
    className: 'fi-icon-illustrative-base'
  },
  {
    attribute: 'fill|stroke',
    value: '#E97025',
    className: 'fi-icon-illustrative-highlight'
  }
]

```

Reply:
This rule change is now committed.